### PR TITLE
Add DNSName flag to scepclient

### PR DESCRIPTION
**Pull request by macmule**
Whilst DNSNames have been referenced in the cert template (https://github.com/micromdm/scep/blob/main/depot/signer.go#L81-L97), neither the scepclient did not have the option to add these to the CSR.

This PR resolves that by adding the option via a dnsname flag.

This could be improved upon though by adding multiple, but a little beyond my 1 days worth of Go experience.

### CODE REVIEW

File: cmd/scepclient/csr.go [hunk 1]
```diff
@@ -19,6 +19,7 @@ const (
 
 type csrOptions struct {
 	cn, org, country, ou, locality, province, challenge string
+	cn, org, country, ou, locality, province, dnsnames, challenge string
```
Comment (jessepeterson) [inline cmd/scepclient/csr.go:21]:
```suggestion
	cn, org, country, ou, locality, province, dnsName, challenge string
```

[mixedCaps](https://go.dev/doc/effective_go#mixed-caps) :) also made this singular as its a single string.

**Review by jessepeterson — CHANGES_REQUESTED** [review]

**Comment by macmule** [comment]
Sorry, made another commit which caught the changes after the mixedCaps miss

**Comment by macmule** [comment]
ran via `gofmt -s` each file flagged with `gofmt -s -l` a few times.. and `gofmt -w`.. now `gofmt -s -l`  has no output.. so fingers crossed this has worked.

**Review by jessepeterson — APPROVED** [review]
lgtm; Thanks!

**Pull request merged**